apps/src/templates/sectionProgress/ProgressViewHeader.jsx

```diff
@@ -19,7 +19,7 @@ class ProgressViewHeader extends Component {
     currentView: PropTypes.oneOf(Object.values(ViewType)),
     sectionId: PropTypes.number.isRequired,
     scriptFriendlyName: PropTypes.string.isRequired,
-    scriptData: scriptDataPropType,
+    scriptData: unitDataPropType,
```

Comment:
I thought this would be a simple find and replace, but I should probably also rename the variables. I think this is still a valuable change, but I can also see wanting to wait to rename until we rename the vars at the same time. What do you think?

Reply:
Yeah...it seems we have two names for this same concept and it's not great. The database table is `scripts`, the Ruby model is `Unit`, and there seems to be a hodgepodge of those two terms used throughout the codebase. Moving in the direction of `Unit` in more places sounds great to me, and doing it everywhere obviously goes beyond the scope of this development task. Pick a stopping point that makes sense to you.